packages/config/config/devices/0x027a/zen16.json

```diff
@@ -0,0 +1,603 @@
+					"value": 0
+				},
+				{
+					"label": "Toggle switch (follow switch)",
+					"value": 1
+				},
+				{
+					"label": "Toggle switch (change state on toggle)",
+					"value": 2
+				},
+				{
+					"label": "Garage door (momentary mMode for Z-Wave control)",
+					"value": 3
+				}
+			]
+		},
+		"3": {
+			"label": "Switch Type for Relay 2 (Sw2)",
+			"description": "Choose the wall switch type you want to connect to the Sw2 terminal.",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 3,
+			"defaultValue": 2,
+			"unsigned": true,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Momentary Switch",
+					"value": 0
+				},
+				{
+					"label": "Toggle switch (follow switch)",
+					"value": 1
+				},
+				{
+					"label": "Toggle switch (change state on toggle)",
+					"value": 2
+				},
+				{
+					"label": "Garage door (momentary mMode for Z-Wave control)",
+					"value": 3
+				}
+			]
+		},
+		"4": {
+			"label": "Switch Type for Relay 3 (Sw3)",
+			"description": "Choose the wall switch type you want to connect to the Sw3 terminal.",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 3,
+			"defaultValue": 2,
+			"unsigned": true,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Momentary Switch",
+					"value": 0
+				},
+				{
+					"label": "Toggle switch (follow switch)",
+					"value": 1
+				},
+				{
+					"label": "Toggle switch (change state on toggle)",
+					"value": 2
+				},
+				{
+					"label": "Garage door (momentary mMode for Z-Wave control)",
+					"value": 3
+				}
+			]
+		},
+		"5": {
+			"label": "LED Indicator Control",
+			"description": "Choose the LED indicator behavior",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 3,
+			"defaultValue": 0,
+			"unsigned": true,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "On when all relays are off",
+					"value": 0
+				},
+				{
+					"label": "On when any relays is on",
+					"value": 1
+				},
+				{
+					"label": "Always off",
+					"value": 2
+				},
+				{
+					"label": "Always on",
+					"value": 3
+				}
+			]
+		},
+		"6": {
+			"label": "Auto Turn-Off Timer for Relay 1",
+			"description": "Sets the time after which you want relay 1 to automatically turn off once it has been turned on.",
+			"valueSize": 4,
+			"unit": "minutes (unless changd by parameter 15)",
+			"minValue": 0,
+			"maxValue": 65535,
+			"defaultValue": 0,
+			"unsigned": true,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true
+		},
+		"7": {
+			"label": "Auto Turn-On Timer for Relay 1",
+			"description": "Sets the time after which you want relay 1 to automatically turn on once it has been turned off.",
+			"valueSize": 4,
+			"unit": "minutes (unless changd by parameter 16)",
+			"minValue": 0,
+			"maxValue": 65535,
```

Comment:
Value size 4 but the maxvalue is only 2 bytes long. Seems weird, is that correct?

Reply:
Also from the manual.